I3uckwheat/cards-of-carousal
# Delete header component

#### 0. Make sure your issue is linked:


closes #221 

#### 1. Purpose:


Now that the HeaderFooterLayout has been created, the Header component has little use. This change deletes the Header component

#### 2. Implementation:


- Delete header component
- Add a styled header, with the same styling, to the remaining components that relied on this header
- Update snapshots

#### 3. Possible Issues:




#### 4. How to Test:


1. Pull any changes and enter my branch
2. Start the frontend and make sure the host layout and settings menu header look normal

#### 5. Cleanup
